ores/features_reverted.py

```diff
@@ -61,8 +61,10 @@ def main():
         rev_pages = read_rev_pages(sys.stdin)
     else:
         rev_pages = read_rev_pages(open(args['--rev-pages']))
-
-    features = import_from_path(args['<features>'])
+    if '<features>' in args:
```

Comment:
Does this work?  "<features>" should always exist, but be None if it is not specified.